src/mumble/Log.cpp

```diff
@@ -54,15 +56,18 @@ LogConfig::LogConfig(Settings &st) : ConfigWidget(st) {
 		twi->setText(ColMessage, messageName);
 		twi->setCheckState(ColConsole, Qt::Unchecked);
 		twi->setCheckState(ColNotification, Qt::Unchecked);
+		twi->setCheckState(ColHighlight, Qt::Unchecked);
 		twi->setCheckState(ColStaticSound, Qt::Unchecked);
 
 		twi->setToolTip(ColConsole, tr("Toggle console for %1 events").arg(messageName));
 		twi->setToolTip(ColNotification, tr("Toggle pop-up notifications for %1 events").arg(messageName));
+		twi->setToolTip(ColHighlight, tr("Toggle window highlight in taskbar (if not active) for %1 events").arg(messageName));
```

Comment:
Taskbar is not a cross-platform term.

Reply:
What about "Application bar"?

Reply:
How about just dropping "in taskbar"? Same for the rest of these.

Reply:
Sounds good.

Every desktop environment has its own way to highlight the window (if supported).